### DISCUSSION TITLE
Starting zwave-js-ui crashes RPi

### DISCUSSION THREAD

**Discussion by brianmichalk1** [zwave-js/zwave-js-ui#3846] [General]
I have a Pi3B+, powered from a 3A usb charger, and the Zooz ZST39LR.  No display manager, strictly console.
I originally installed about six months ago, and everything was perfect.  I had corruption on my cheap SDCard, so pulled down the machine and made backups.
I reinstalled using my bash history as a guide, and restored the zwave configuration directory.
When I start the UI, the machine crashes.  No pings, nothing.  With a monitor attached, the console is dead.  The green led remains solid for a while and eventually turns off.
I reboot, and start from a clean zwave directory.  same thing.
I am beginning to suspect something wrong with the stick.  When I connect the stick, I get this, which looks normal.
```
[   68.053224] usb 1-1.2: new full-speed USB device number 6 using dwc_otg
[   68.200612] usb 1-1.2: New USB device found, idVendor=1a86, idProduct=55d4, bcdDevice= 4.43
[   68.200645] usb 1-1.2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[   68.200660] usb 1-1.2: Product: 800 Z-Wave Stick
[   68.200672] usb 1-1.2: Manufacturer: Zooz
[   68.200683] usb 1-1.2: SerialNumber: 533D004242
[   68.202598] cdc_acm 1-1.2:1.0: ttyACM0: USB ACM device
```
Is there a linux app I can install that checks the health of the stick?

**Comment by brianmichalk1** [comment]
I should add that I can start the zwave-js-ui without the path to the stick, and everything works as expected.  If I change the settings via web interface, and click "save", the Pi crashes.
If I have the settings.json set up with the path to the stick, and start on the CLI, the program crashes at the point where it says it's opening the device.

**Comment by kpine** [comment]
There is an existing issue for this RPi3 problem. https://github.com/zwave-js/zwave-js-ui/issues/3639

**Reply by brianmichalk1** [reply]
That looks exactly like my problem.  Thanks.
